MAINTENANCE.md

```diff
@@ -163,3 +164,46 @@ the build directory you have to run CMake again manually.
 
 Guides for debugging GMT are provided in the
 [developer resources section](https://docs.generic-mapping-tools.org/dev/debug.html) of the GMT documentation.
+
+## Using build and test aliases
+
+The file [admin/bashrc_for_gmt](/admin/bashrc_for_gmt) contains useful aliases for building and testing GMT that some
+developers chose to use. New pull requests with other aliases that you find helpful are welcome. This file is version controlled, so you should copy the file to a different location in order to edit and use it. For example, use these
+commands to copy it to your home directory:
+
+```
+cd <path-to-gmt>
+cp admin/bashrc_for_gmt ~/bashrc_for_gmt
+```
+
+Here are the steps for setting up bashrc_for_gmt after copying it to a new location:
+
+- If you do not have [ninja](https://ninja-build.org/) installed, you will need to change `builder=ninja` to
+  `builder=make` and `Bname="Ninja"` to `Bname="Unix Makefiles"`. It is recommended that you use ninja.
+- You may need to update `pngview=open` and `pngview=open` depending on your preferred program for viewing files.
+- Optionally, change `ncores=4` to the number of cores to use for building and running tests.
+- Change `MATLAB=/Applications/MATLAB_R2019a.app` to the path for your version of the MATLAB app.
+- Set `REPO_DIR` to the path that contains the GMT repository.
+- Set `DATA_DIR` to the path that contains the folders `dcw-gmt-1.1.4/` and `gshhg-gmt-2.3.7/` for the dcw and gshhg
+  dataset respectively. If these folders are not located in the same path, you can instead delete L25
+  (`DATA_DIR=<path to directory containing GSHHG and DCW>`) and set the individual paths to the GSHHG and DCW source
+  in L30 (`export GMT_GSHHG_SOURCE=${DATA_DIR}/gshhg-gmt-2.3.7`) and
+  L31(`export GMT_DCW_SOURCE=${DATA_DIR}/dcw-gmt-1.1.4`).
+- Edit the file `~/.bashrc` to include the line `source <path>/bashrc_for_gmt`. If you set up `bashrc_for_gmt` in your
+  home directory, this line should be `source ~/bashrc_for_gmt`.
```

Comment:
Wondering if this bullet point should probably be on the top of the list. Or if it's better to have people configure things properly first.

Reply:
I think setting up first is better.